modules/clickhouse/src/test/java/org/testcontainers/ClickhouseTestImages.java

```diff
@@ -3,5 +3,5 @@
 import org.testcontainers.utility.DockerImageName;
 
 public interface ClickhouseTestImages {
-    DockerImageName CLICKHOUSE_IMAGE = DockerImageName.parse("clickhouse/clickhouse-server:21.9.2-alpine");
+    DockerImageName CLICKHOUSE_IMAGE = DockerImageName.parse("clickhouse/clickhouse-server:24.8.12.28");
```

Comment:
I think it is worth to test against 21.9.2-alpine. That way any related change in the future will allow us to detect a regression.

Reply:
Looks like `org.testcontainers.ClickhouseTestImages` has changed. Opening a PR has never been easier.